### ISSUE TITLE
Creating isochrones with osrm

### ISSUE THREAD

**Issue by psksBrutus** [Project-OSRM/osrm-backend#401]
Hello everyone, 
would it be possible to invent a plugin for the actual osrm project which generates isochrones on the fly ?
Or even better ist there someone who has already implemented this feature in his repo?

Isochrones - http://wiki.openstreetmap.org/wiki/Isochrone

Thanks for your answers!

**Comment by TheMarex** [comment]
There is a new PR https://github.com/Project-OSRM/osrm-backend/pull/2477 that implements this but is blocked by some architecture problems.
